cirq-google/cirq_google/optimizers/convert_to_sycamore_gates.py

```diff
@@ -151,8 +151,8 @@ def optimization_at(
             next_index = circuit.next_moment_operating_on(op.qubits, index + 1)
             if next_index is not None:
                 ops_in_front = list({circuit.operation_at(q, next_index) for q in op.qubits})
-                if len(ops_in_front) == 1 and isinstance(ops_in_front[0], cirq.GateOperation):
-                    gate2 = ops_in_front[0].gate
+                if len(ops_in_front) == 1 and ops_in_front[0]:
+                    gate2 = cast(cirq.Operation, ops_in_front[0]).gate  # For mypy.
```

Comment:
Is this `cast` required? mypy should recognize `ops_in_front[0]` as an `Optional[cirq.Operation]` based on the return type of `Circuit.operation_at`, and the if-block verifies that it's non-None.